project/frontend/src/components/UserPreferenceForm.js

```diff
@@ -214,86 +226,41 @@ function PreferenceForm(props) {
-          <img src={Cuisine} alt='' />
-          <label htmlFor='cuisine'>Cuisines</label>
-          <CuisineAutocomplete
-            cuisineOptions={cuisineOptions}
-            setCuisine={setCuisine}
-          />
-        </div>
-        <div className='preference-form-row'>
-          <div className='preference-form-column'>
-            <div className='preference-form-row'>
-              <img src={Distance} alt='' />
-              <label htmlFor='radius'>Distance</label>
-              {getSelect('radius', radius, setRadius, distancesInMiles)}
-            </div>
-            <div className='preference-form-row'>
-              <img src={Experience} alt='' />
-              <label htmlFor='diningExp'>Experience</label>
-              {getSelect(
-                'diningExp',
-                diningExp,
-                setDiningExp,
-                diningExperiences
-              )}
-            </div>
-            <div className='preference-form-row'>
-              <img src={Price} alt='' />
-              <label htmlFor='priceLevel'>Price Level</label>
-              {getSelect('priceLevel', priceLevel, setPriceLevel, prices)}
-            </div>
-          </div>
-          <div className='preference-form-column'>
-            <div className='preference-form-row'>
-              <label>Importance</label>
-              {getSlider(radiusWeight, setRadiusWeight, !radius)}
-            </div>
-            <div className='preference-form-row'>
-              <label>Importance</label>
-              {getSlider(diningExpWeight, setDiningExpWeight, !diningExp)}
-            </div>
-            <div className='preference-form-row'>
-              <label>Importance</label>
-              {getSlider(priceLevelWeight, setPriceLevelWeight, !priceLevel)}
-            </div>
-          </div>
-        </div>
-        <div
-          className='preference-form-row'
-          style={{ justifyContent: 'center', margin: '32px 0px' }}>
-          <label htmlFor='open'>Open Now</label>
-          <input
-            name='open'
-            type='checkbox'
-            checked={open}
-            onChange={(event) => setOpen(event.target.checked)}
-          />
-        </div>
-        <div className='preference-form-submit-container'>
-          <button type='submit'>Find my match</button>
-        </div>
-      </form>
+      <h4>
+        Your preferences.
+        <StyledTooltip title={tooltipInfo} interactive>
+          <Info />
+        </StyledTooltip>
+      </h4>
+      <p>
+        Please enter your restaurant preferences below. You may leave any field
+        blank if you have no preference. Specify an importance to indicate your
+        priority for different fields.
+      </p>
+      <PreferenceForm
+        rowItemLabels={{
+          cuisine: 'Cuisines',
+          location: 'Location',
+          experience: 'Experience',
+          price: 'Price Level',
+          distance: 'Distance',
```

Comment:
nit: put these in a variable with the other vars (distancesInMiles, prices etc) above